Bio/Blast/NCBIWWW.py

```diff
@@ -42,13 +46,18 @@ def qblast(program, database, sequence, url_base=NCBI_BLAST_URL,
            ):
     """BLAST search using NCBI's QBLAST server or a cloud service provider.
 
-    Supports all parameters of the qblast API for Put and Get.
+    Supports all parameters of the old qblast API for Put and Get.
 
-    Please note that BLAST on the cloud supports the NCBI-BLAST Common
-    URL API (http://ncbi.github.io/blast-cloud/dev/api.html). To
-    use this feature, please set url_base to
-    'http://host.my.cloud.service.provider.com/cgi-bin/blast.cgi' and
-    format_object='Alignment'. For more details, please see
+    Please note that NCBI uses the new Common URL API for BLAST searches
+    on the internet (http://ncbi.github.io/blast-cloud/dev/api.html). Thus,
+    some of the parameters used by this function are not or not longer
```

Comment:
Something odd with the wording here, "are not are not longer officially supported".

Maybe just "are no longer officially supported" is close to your intent?

Reply:
"Are not *or* not longer" ("are not/not longer supported") to say that some of them were supported in the past (old qblast) but some of them (e.g. SHORT_QUERY_ADJUST) have never been officially supported (or documented)

Reply:
Sorry, I missed the "or" in the earlier quote.

How about "... are not (or are no longer) officially supported." here?

Reply:
OK.